src/content/1.7/modules/sell/techvalidation-checklist.md

```diff
@@ -34,6 +34,8 @@ Additionally, distribution licenses like CC-0 or CC-by-sa are appropriate for ar
 
 Module may create all the table they need in the database. However altering core table is forbidden.
 
+In case you want to add columns for a existing object, the workaround is to create a new table with a foreign key targetting the primary key of the core table.
```

Comment:
https://www.a-or-an.com/a_an/existing :)

Isn't it a "new field", not "new column" for Object?

Reply:
Thanks for the feedback, I fixed the typo.

In this case we are talking about the database and not ObjectModels, that's why I was talking about `columns`

Reply:
Edited text looks better 👍